format.go

```diff
@@ -40,7 +42,9 @@ type Node interface {
 	Size() (uint64, error)
 }
 
+// NodeGetter is the interface that wraps the basic get method
```

Comment:
NodeGetter is really just a read-only DAG service. I'd take a look at #8, it makes a lot of this clearer. Actually, I recommend making the pull request against that branch as I'll likely merge that first.